Public/Sdk/SelfHost/MacOS/xcodebuild.dsc

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft. All rights reserved.

```

Comment:
exactly the same as before, just moved to parent directory and enclosed in `export namespace XCode`